homeassistant/runner.py

```diff
@@ -49,13 +51,46 @@ class RuntimeConfig:
     open_ui: bool = False
 
 
+def can_use_pidfd() -> bool:
```

Comment:
I'm not sure there is much value in adding a tests for this since its a line for line backport from 3.12